src/main/java/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Error reading file: " + filePath, e);
+        }
+    }
+}
```

Comment:
The class does not use try-with-resources, but since Files.readAllLines does not require an explicit resource to be closed in this context, it's fine here.